base/multidimensional.jl

```diff
@@ -1298,9 +1298,16 @@ hash(x::Prehashed) = x.hash
 
 Returns an array containing only the unique elements of the iterable `itr`, in
 the order that the first of each set of equivalent elements originally appears.
+The unicity is determined by [`isequal`](@ref).
```

Comment:
For reference to those interested: https://english.stackexchange.com/questions/225094/uniqueness-vs-unicity